cmd/syncthing/cli/main.go

```diff
@@ -150,20 +150,16 @@ func Run() error {
 		Usage: "Syncthing command line interface",
 		Flags: fakeFlags,
 		Subcommands: []cli.Command{
-			{
-				Name:        "config",
-				HideHelp:    true,
-				Usage:       "Configuration modification command group",
-				Subcommands: commands,
-			},
+			configCommand,
 			showCommand,
 			operationCommand,
 			errorsCommand,
+			debugCommand,
 		},
 	}}
 
-	tty := isatty.IsTerminal(os.Stdin.Fd()) || isatty.IsCygwinTerminal(os.Stdin.Fd())
-	if !tty {
+	// tty := isatty.IsTerminal(os.Stdin.Fd()) || isatty.IsCygwinTerminal(os.Stdin.Fd())
+	if false {
```

Comment:
???

Reply:
Lol, my git foo: When running in delve/debugger, this terminal business fails. I was too lazy to look into what the purpose of it all is and ended up committing the workaround.